en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-configuration.md

```diff
@@ -0,0 +1,97 @@
+    
+    <img src="../../../../assets/img/connectors/bigquery-credentials-page.png" title="Bigquery-credentials-page" width="600" alt="Bigquery-credentials-page"/>   
+   
+3. If you do not already have a project, you create a new project, click **Create credentials** and then select **OAuth client ID**.     
+   
+    <img src="../../../../assets/img/connectors/bigquery-create-credentials.png" title="Select OAuth client ID" width="600" alt="Select OAuth client ID"/> 
+   
+4. Next, **select** Web Application, and **create a client**.
+  
+    <img src="../../../../assets/img/connectors/bigquery-select-web-application.png" title="Select web application" width="600" alt="Select web application"/> 
+
+5. Add [https://www.google.com](https://www.google.com) as the redirect URI (You can add any URI which you need as redirect URI) under **Authorized redirect URIs**, and then click **Create**. This displays the **client ID** and **client secret**.
+
+    <img src="../../../../assets/img/connectors/biguery-authorization-redirect-URI.png" title="Authorization-redirect-URI" width="600" alt="Authorization-redirect-URI"/> 
+   
+6. Make a note of the **client ID** and **client secret** that is displayed, and then **click OK**.   
+   
+7. Click **Library** on the left navigation pane.
+
+    <img src="../../../../assets/img/connectors/bigquery-select-library.png" title="Select library" width="600" alt="Select library"/> 
+
+8. Search  **BigQuery API** under the **Big data category**.
+  
+    <img src="../../../../assets/img/connectors/bigquery-select-api.png" title="Bigquery API" width="600" alt="Pubsub API"/> 
+
+9. Click **Enable**. This enables the BigQuery API.   
+   
+    <img src="../../../../assets/img/connectors/bigquery-enable-api.png" title="Bigquery enable API" width="600" alt="Pubsub enable API"/> 
+    
+10. Get the authorization code by sending a GET request to the following URL. Replace the `<redirect_uri>` and `<client_ID>` with the redirect uri and client ID values noted in the previous steps. Enter the following URL in your web browser.
+ 
+    ```
+    https://accounts.google.com/o/oauth2/auth?redirect_uri=<redirect_uri>&response_type=code&client_id=<client_ID>&scope=https://www.googleapis.com/auth/bigquery&approval_prompt=force&access_type=offline
+    ```
+    Note the authorization code for future use.
+    
+    <img src="../../../../assets/img/connectors/bigquery-get-authorization-code.png" title="Get authorization code" width="600" alt="Get authorization code"/> 
+    
+11. Get the `access token` and `refresh token` by sending a POST request to the URL given below. Be sure to use an **x-www-form-urlencoded** body with the `<authorization_code>`, `<client_id>`, `<client_secret>`, and `<redirect_uri>` values noted before, and also set the `grant_type` to **authorization_code**. Note the access token and refresh token for future use. 
```

Comment:
You will need them to configure WSO2 EI Big Query Connector.

Reply:
Updated.